src/main/java/frc/robot/Robot.java

```diff
@@ -58,6 +62,8 @@ public Robot() {
 		this.elbow = new Elbow(ElbowFactory.create(ElbowConstants.LOG_PATH));
 		BrakeStateManager.add(() -> elbow.setBrake(true), () -> elbow.setBrake(false));
 		this.funnel = new Funnel(FunnelFactory.create(FunnelConstants.LOG_PATH));
+		this.roller = new Roller(RollerFactory.create(RollerConstants.LOG_PATH));
+		BrakeStateManager.add(() -> roller.setBrake(true), () -> roller.setBrake(false));
 		configureBindings();
```

Comment:
```suggestion
		this.roller = new Roller(RollerFactory.create(RollerConstants.LOG_PATH));
		BrakeStateManager.add(() -> roller.setBrake(true), () -> roller.setBrake(false));
		
		configureBindings();
```
additional space to separate two different things